### DISCUSSION TITLE
Learn Medusa now, or wait for 2.0

### DISCUSSION THREAD

**Discussion by Tobbe** [medusajs/medusa#6503] [Q&A]
I run an e-comm store that I've built all on my own. Fully custom backend + frontend. It's a lot of work to maintain, so I was considering switching to Medusa. But I see you're well on your way to a big new 2.0 release. 

Is it worth it to learn Medusa now, or should I wait for 2.0 to be released? 
Or, asked in a different way: If I learn Medusa now, how much of that knowledge will transfer over to Medusa 2.0?

**Comment by predaytor** [comment]
Just started a project with Medusa (Fly.io, Neon Postgres, Upstash Redis). Can't imagine building something like that myself. 

Shopify (Hydrogen) is a premium e-commerce solution, but it's more for high-scale products. The Medusa API is great. Can't wait for Vite integration in Medusa 2.0. I don't think there will be many critical changes, if so it would be trivial to move over a few days.

**Reply by Tobbe** [reply]
> Can't imagine building something like that myself.

Haha. Yeah, it was a lot of work! :D

**Comment by LucasMatuszewski** [comment]
Very good question, UP :)
And when will it be probably released? ;)

I would also ask if it's possible to easily integrate the existing Next.js FE App with Medusa in the future? (in my case we need only marketing Landing for now, can add eCommerce within 2-3 months)

Or is it much better/easier to start with the Medusa template now (even in the v1 version) and then migrate to V2?

**Comment by dpdenton** [comment]
Also very keen to know when v2 will be released (even alpha/preview version)

**Reply by Matb85** [reply]
Bump, thinking of starting a project based on Medusa JS, should I start now and then upgrade or wait a few weeks for v2 alpha/beta?
